OpenDocumentReader/DocumentViewController.swift

```diff
@@ -443,6 +444,7 @@ class DocumentViewController: UIViewController, DocumentDelegate, GADBannerViewD
             self.webview.loadFileURL(doc.fileURL, allowingReadAccessTo: doc.fileURL)
             
             progressBar.isHidden = true
+            searchButton.isEnabled = false
```

Comment:
we never need to set it to true again because DocumentViewController is recreated for each document that is opened, right?

Reply:
That is correct